CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 Queries with the less `<` or greater `>` operators would sometimes produce
+  incorrect results for the `duration` type. The results should now be correct.
```

Comment:
"should now be correct" doesn't sound very convincing. Let's be more confident and write this as:

```
- 🐞 Queries with the less `<` or greater `>` operators produced off-by-one
  results for the `duration` when the query contained a finer resolution than the
  index. The operator now works as expected.
```

Reply:
done